Expecto.Tests/Tests.fs

```diff
@@ -124,8 +123,8 @@ let tests =
             let greenStart = ANSIOutputWriter.colourText (ANSIOutputWriter.getColour()) ConsoleColor.Green
             let greenEnd = ANSIOutputWriter.colourText (ANSIOutputWriter.getColour()) ConsoleColor.Cyan
             "Failing - record with different content.\nRecord does not match at position 2 for field named `b`. Expected field with value: \"dw\", but got \"de\".\n"+
-            greenStart+"expected"+greenEnd+":\n{a = \"dd\";\n b = \"d"+greenStart+"w"+greenEnd+"\";}\n"+
-            redStart+"  actual"+redEnd+":\n{a = \"dd\";\n b = \"d"+redStart+"e"+redEnd+"\";}"
+            greenStart+"expected"+greenEnd+":\n{ a = \"dd\"\n  b = \"d"+greenStart+"w"+greenEnd+"\" }\n"+
+            redStart+"  actual"+redEnd+":\n{ a = \"dd\"\n  b = \"d"+redStart+"e"+redEnd+"\" }"
```

Comment:
This is something that I could not track why I had to change. The difference is the following:

The previous version printed in this format here:
```fs
{a = "dd";
 b = "dw";}
  actual:
{a = "dd";
 b = "de";}
```

However, the new one expects this format:
```fs
{ a = "dd"
  b = "de" }
```

Reply:
As far as I can tell, yes. It does use `%A`... Is there a better approach here?

Reply:
that's a fine approach! it just also explains why the format differed. you're essentially off-loading the responsibility for formatting, so you just might have to change your test expectations every so often :shrug: